src/components/tasks/card/SuggestionBox.tsx

```diff
@@ -0,0 +1,107 @@
+import { FC, useEffect, useState } from 'react';
+import { DUMMY_PROFILE as placeholderImageURL } from '@/constants/display-sections';
+import classes from '@/components/tasks/card/suggestion.module.scss';
+import fetch from '@/helperFunctions/fetch';
+import { userDataType } from '@/interfaces/user.type';
+
+type Props = {
+    assigneeName: string;
+    showSuggestion: boolean;
+    onClickName: (userName: string) => void;
+};
+
+type GithubInfo = {
+    github_id: string;
+    url: string;
+};
+
+const SuggestionBox: FC<Props> = ({
+    assigneeName,
+    showSuggestion = false,
+    onClickName = () => undefined,
+}) => {
+    const [loading, setLoading] = useState<boolean>(false);
+    const [suggestions, setSuggestions] = useState<GithubInfo[]>([]);
+
+    const clickHandler = (e: any) => {
+        onClickName(e.target.innerText);
+    };
+
```

Comment:
you cannot use e of type any, if we need to use any then what is the use of typescript, we could have just used JS

Reply:
fixed @Pratiyushkumar